packages/react-input/src/components/Input/useInput.ts

```diff
@@ -30,8 +30,8 @@ export const useInput = (props: InputProps, ref: React.Ref<HTMLElement>): InputS
       input: 'input',
     },
     // temporarily must add fake children to prevent getSlots from substituting nullRender
-    input: resolveShorthand(props.input, { children: '😭' }),
-    inputWrapper: resolveShorthand(props.inputWrapper, { children: '😭' }),
+    input: resolveShorthand(props.input, { optional: false }),
+    inputWrapper: resolveShorthand(props.inputWrapper, { optional: false }),
```

Comment:
This looks kinda strange as we need to override a default value to false. Did you consider naming it `required`?

```suggestion
    input: resolveShorthand(props.input, { required: true }),
    inputWrapper: resolveShorthand(props.inputWrapper, { required: true }),
```

Reply:
I did not consider that. Yeah, it's possible 👍🏼, I'll see the changes